pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/PostStageTest.java

```diff
@@ -121,7 +121,7 @@ public void withAllLocalChanged() throws Exception {
     @Test
     public void failingNotifications() throws Exception {
         expect(Result.FAILURE, "failingNotifications")
-                .logContains("hello", "Post stage", "farewell", "Error in stage post:", "I AM HERE STILL")
+                .logContains("hello", "Post stage", "farewell", "I AM HERE STILL")
```

Comment:
This test doesn't really test that a failing notification is communicated any more.